assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -16,14 +16,58 @@ import { useEffect } from '@wordpress/element';
 import UploadButton from '../uploadButton';
 import useLibrary from './useLibrary';
 
-const Image = styled.img`
+const ThumbList = styled.div`
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+`;
+
+const Thumb = styled.div`
+	position: relative;
 	height: 150px;
 	width: 48%;
 	padding: 3px;
 	margin: 3px;
 	border: 1px solid white;
 `;
 
+const ThumbButtons = styled.div`
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: 0;
+	height: 50px;
+	z-index: 2;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: center;
+	background: rgba(0, 0, 0, 0.2);
+
+	opacity: 0;
+  ${ Thumb }:hover & {
+    opacity: 1;
+  }
+`;
+
+const ThumbButton = styled.button`
+	background: rgba(0, 0, 0, 0.7);
+	color: #fff;
```

Comment:
Should these come from the theme colors? Or is this temporary, waiting for the final UX anyway?

Reply:
Rolling back this file, but asked on the Slack how to use theme colors. I wasn't sure if I understood it right...